bokehjs/src/lib/models/annotations/legend.ts

```diff
@@ -131,9 +131,10 @@ export class LegendView extends AnnotationView {
     return {x: sx, y: sy, width: legend_width, height: legend_height}
   }
 
-  bbox(): BBox {
```

Comment:
Having access to the bbox is actually useful to me.

Reply:
OK well the issue was that the `BBox` was being used to determine whether to engage an interactive feature or not, and since there is no "null bbox" there was no clean way to turn that off under certain conditions. I will: 

* restore an `interactive_bbox` function, that just unconditionally returns a bbox that describes what the interactive hit area should be, if interactions are turned on

* rename `bbox_contains` to `interactive_hit` and have it use the above and be responsible for the on/off policy setting

Reply:
fixed in https://github.com/bokeh/bokeh/pull/7839/commits/adfa5840cee963976f522a471c18b3eaab404e71